FlowCrypt/Models/Realm Models/ClientConfigurationRealmObject.swift

```diff
@@ -19,6 +19,8 @@ final class ClientConfigurationRealmObject: Object {
     @Persisted var disallowAttesterSearchForDomains: Data?
     @Persisted var enforceKeygenAlgo: String?
     @Persisted var enforceKeygenExpireMonths: Int
+    @Persisted var disallowPasswordMessagesForTerms: Data?
+    @Persisted var disallowPasswordMessagesErrorText: String?
```

Comment:
looks like migration is missing, as I've got such error on app startup:

<img src="https://github.com/user-attachments/assets/8f2a2295-115f-459c-b700-73d363e2bca3" width="300">

Reply:
Yep, Right